extensions/dependencies/guppy.html

```diff
@@ -8,3 +8,27 @@
     '/third_party/static/guppy-be6f14/src/transform.xsl',
     '/third_party/static/guppy-be6f14/src/symbols.json');
 </script>
+
+<style>
+    .guppy_active {
+      box-shadow: 0 0 5px rgba(81, 203, 238, 1) !important;
```

Comment:
Oh I did not notice this file. I though the inline style elements were being set in guppy.js. But there is this one style that is set in guppy.js - `this.editor.style.boxShadow = "1px 1px 1px 0 lightgray inset";`. I don't think I'll be able to overcome this right? I can add a new element to wrap guppy and a bit of Javascript to change the box shadow when the textbox is clicked, but it might be a bit of overkill. So what do you think I should do?